.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @rungalileo/platform
```

Comment:
What's the diff between this and the teams in GitHub Settings?

Reply:
This sets the reviewer group to auto-assign and who can approve. The GH settings sets permissions for access.